### DISCUSSION TITLE
병원 한곳만 예약하는것 말고 지도로하는건 막혔나요?

### DISCUSSION THREAD

**Discussion by loojods** [changdoc/naver-vaccine-macro#208] [Q&A]
![image](https://user-images.githubusercontent.com/88393569/128046510-95684238-e02b-4840-b4d7-0ca676811547.png)
이 창으로 한곳 병원 예약하는것 말고 지도로 여러병원 하는방법은 막혔나요??? 감사합니다 !

**Comment by changdoc** [comment]
지도에서 동작 방법은 Isseus 메뉴를 참고하시면 됩니다.
